### ISSUE TITLE
Asterisk in allowlist not work

### ISSUE THREAD

**Issue by Iowerth** [AdguardTeam/AdguardBrowserExtension#2053]
### Issue Details

Notice for my case: allowlist is inverted. Ads are blocked only on websites added to it. Issue I describe appeared in 4.0.141.

- **AdGuard version:**
  - 4.0.161
- **Browser and version:**
  - Firefox 99.0.1
- **Operating system and version:**
  - Windows 10 version 1909

### Expected Behavior

If string `*.domain.com` added to allowlist then ads are blocked on `subdomain1.domain.com` and `subdomain1.domain.com`

### Actual Behavior

If string `*.domain.com` added to allowlist then ads are **_not_** blocked on `subdomain1.domain.com` and `subdomain1.domain.com`. I have to add these subdomains to allowlist manually.

**Comment by maximtop** [comment]
Closing this as a duplicate of https://github.com/AdguardTeam/AdguardBrowserExtension/issues/2020
